src/integrationTest/java/uk/nhs/hee/tis/trainee/forms/TestJwtUtil.java

```diff
@@ -70,15 +61,28 @@ public static String generateTokenForTisId(String traineeTisId) {
    */
   public static String generateTokenForTrainee(String traineeTisId, String email, String givenName,
       String familyName) {
+    String features = String.format("{\"ltft\":true,"
+        + "\"ltftProgrammes\":[\"%s\"]}", FEATURES_LTFT_PROGRAMME);
     String payload = String.format("{\"%s\":\"%s\"", TIS_ID_ATTRIBUTE, traineeTisId)
         + (email == null ? "" : String.format(",\"%s\":\"%s\"", EMAIL_ATTRIBUTE, email)
         + (givenName == null ? "" : String.format(",\"%s\":\"%s\"", GIVEN_NAME_ATTRIBUTE, givenName)
         + (familyName == null ? "" : String.format(",\"%s\":\"%s\"", FAMILY_NAME_ATTRIBUTE,
-        familyName))))
+        familyName))
+        + String.format(",\"%s\":%s", FEATURES_ATTRIBUTE, features)))
         + "}"; // :tears:
     return generateToken(payload);
   }
 
+  /**
+   * Generate a token with default values for the various attributes as the payload.
+   *
+   * @param traineeTisId The TIS ID to inject in to the payload.
+   * @return The generated token.
+   */
+  public static String generateTokenForTrainee(String traineeTisId) {
```

Comment:
This is a bit clunky, but replaces the 'generateTokenForTisId()' that various tests were using, to ensure that missing features don't either break tests or mask negative tests.